src/reanimated2/layoutReanimation/sharedTransitions/SharedTransition.ts

```diff
@@ -10,6 +10,7 @@ import {
   SharedTransitionType,
 } from '../animationBuilder/commonTypes';
 import type { StyleProps } from '../../commonTypes';
+import { ReduceMotion } from '../../commonTypes';
```

Comment:
```suggestion
import type { ReduceMotion } from '../../commonTypes';
```

Reply:
This won't work since `ReduceMotion` is used as a value in this file

Reply:
ah, I forgot it was this enum 😅